client/post-editor/media-modal/detail/detail-item.jsx

```diff
@@ -144,15 +171,28 @@ class EditorMediaModalDetailItem extends Component {
 		);
 	}
 
-	renderImageEditorButtons( item, classname = 'is-desktop' ) {
-		const { site } = this.props;
-
-		if ( ! enableImageEditing( item, site ) ) {
+	renderMediaEditorButtons( item, classname = 'is-desktop' ) {
+		if ( ! item ) {
 			return null;
 		}
 
+		const mimePrefix = MediaUtils.getMimePrefix( item );
 		const classes = classNames( 'editor-media-modal-detail__edition-bar', classname );
 
+		if ( 'video' === mimePrefix ) {
+			return this.renderVideoEditorButtons( item, classes );
+		}
+
+		return this.renderImageEditorButtons( classes, classname );
```

Comment:
fun fact! we can simplify this here

```js
return 'video' === mimePrefix
	? this.renderVideoEditorButtons( … )
	: this.renderImageEditorButtons( … )
```

it's my personal preference as well because of the way it brings the two options into visual alignment vs leaving them as separate structures